modules/social_features/social_topic/tests/src/Kernel/GraphQL/QueryTopicsTest.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+namespace Drupal\Tests\social_topic\Kernel\GraphQL;
+
+use Drupal\node\NodeInterface;
+use Drupal\Tests\node\Traits\NodeCreationTrait;
+use Drupal\Tests\social_graphql\Kernel\SocialGraphQLTestBase;
+use Drupal\Tests\user\Traits\UserCreationTrait;
+
+/**
+ * Tests the topics field on the Query type.
+ *
+ * @group social_graphql
+ * @group social_topic
+ */
+class QueryTopicsTest extends SocialGraphQLTestBase {
+
+  use NodeCreationTrait;
+  use UserCreationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'social_topic',
+    'entity',
+    // For the topic author and viewer.
+    'social_user',
+    'user',
+    // User creation in social_user requires a service in role_delegation.
+    "role_delegation",
+    // social_comment configures topics for nodes.
+    'node',
+    // The default topic config contains a body text field.
+    'field',
+    'text',
+    'filter',
+    'file',
+    'image',
+    // For the comment functionality.
+    'social_comment',
+    'comment',
+    'menu_ui',
```

Comment:
Check configuration.

Reply:
There is dependencies in `node.type.topic.yml` file, so not sure if we can scrap it
dependencies:
  module:
    - menu_ui

Reply:
```suggestion
    // node.type.topic has a configuration dependency on the menu_ui module.
    'menu_ui',
```

Reply:
Added